### ISSUE TITLE
Restore originally focused element when BytmDialog closes

### ISSUE THREAD

**Issue by Sv443** [Sv443/BetterYTM#178]
For example when the config menu is open and the "feature help" dialog is opened on top and then closed, the focus is reset back to the first element matching `#bytm-cfg-menu *[tabindex="0"]`, instead of the (?)-button that was clicked to bring up the help dialog.  
This is likely due to every other part of the page being set to [`inert`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/inert), so I doubt there is an elegant way of working around this without a) reworking the config menu to use the [native `<dialog>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/dialog), or b) keeping track of each element's [`focusout`](https://developer.mozilla.org/en-US/docs/Web/API/Element/focusout_event), then conditionally restoring focus to said element.  
The exact behavior should be configurable under the new "Accessibility" feature group.
